static_config.yaml

```diff
@@ -5,14 +5,6 @@
 edx_s3_processed_prefix: prod-edx/processed/
 edx_s3_rejected_prefix: prod-edx/rejected/
 
-# Celery Info
-onsite_worker: False
-celery_threads: 1
-celery_app_name: veda_production
-celery_worker_queue: encode_worker,large_encode_worker
-celery_deliver_queue: deliver_worker
-celery_heal_queue: heal_queue
-
```

Comment:
why we have deleted all these? this will cause crashes for new sandboxes because those sandboxes expect these settings in this file.

Reply:
Yep! I was trying to move these settings, but point taken. Replacing.